packages/sdk-v2/src/sdk/onboard/modules/utils.ts

```diff
@@ -0,0 +1,26 @@
+import bowser from 'bowser'
```

Comment:
please move to common utils as I've asked 
`packages/sdk-v2/src/sdk/base/utils`